cmd/blocks/cmd.go

```diff
@@ -0,0 +1,69 @@
+package blocks
+
+import (
+	"errors"
+	"fmt"
+
+	errs "github.com/onflow/flow-evm-gateway/models/errors"
+	"github.com/onflow/flow-evm-gateway/storage/pebble"
+	"github.com/rs/zerolog/log"
+	"github.com/spf13/cobra"
+
+	flowGo "github.com/onflow/flow-go/model/flow"
+)
+
+var Cmd = &cobra.Command{
+	Use:   "check-blocks-integrity",
+	Short: "Checks the EVM blocks integrity",
+	RunE: func(*cobra.Command, []string) error {
+		if databaseDir == "" || chain == "" {
+			return fmt.Errorf("databaseDir, chain flags must be provided")
+		}
```

Comment:
This could be replacep with 
```
Cmd.MarkFlagRequired("database-dir")
Cmd.MarkFlagRequired("chain")
```
in the init function

Reply:
Good one :ok_hand: 
Added in https://github.com/onflow/flow-evm-gateway/pull/775/commits/ebe41ad5e04e160b621e7a355dad5058283e8692